src/main/java/com/commands/FireRocketCommand.java

```diff
@@ -22,10 +22,15 @@ public FireRocketCommand(String direction) {
 	
 	public void execute(Vehicle vehicle) {
 		// TODO: set Positions
+		if (!vehicle.isRocketReady()) {
+			return;
+		}
 		World world = vehicle.getWorld();		
 		Rocket rocket  = new Rocket (this.direction);
-		rocket.setTargetSpeed(100);
+		//TODO speed of rocket
+		rocket.setTargetSpeed(300);
 		rocket.setPosition(vehicle.getPosition());
+		//rocket.setCurrentRoadpiece(vehicle.getCurrentRoadpiece());

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)